From 04b95611da92bb9a1e1961d604f0fc148e80e4fd Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Thu, 16 Nov 2017 11:05:14 -0800
Subject: [PATCH] ZAPI-812 Unable to provision kvm with vmapi version
 master-20171110T184239Z-ga60a380 Reviewed by: Josh Wilsdon
 <jwilsdon@joyent.com> Approved by: Josh Wilsdon <jwilsdon@joyent.com>

---
 lib/workflows/provision.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/workflows/provision.js b/lib/workflows/provision.js
index 0cd988e..7bd6c05 100644
--- a/lib/workflows/provision.js
+++ b/lib/workflows/provision.js
@@ -372,7 +372,7 @@ function ensureImage(job, cb) {
     var ensurePayload = {};
 
     if (job.params['brand'] === 'kvm') {
-        ensurePayload.image_uuid = job.params.payload.disks[0].image_uuid;
+        ensurePayload.image_uuid = job.params.disks[0].image_uuid;
     } else {
         ensurePayload.image_uuid = job.params.image_uuid;
     }
-- 
2.21.0

